source/api/plugins/browser-launch-api.md

```diff
@@ -96,22 +111,18 @@ module.exports = (on, config) => {
       // for example, to set `somePreference: true` in Preferences:
       launchOptions.preferences.default.somePreference = true
 
-      // more information about Chromium preferences can be found here:
-      // https://www.chromium.org/developers/design-documents/preferences
```

Comment:
This is a very confusing doc, and I have a hard time understanding how it is relevant to sending preferences. Although I cannot figure out the signature for sending any perfs in https://src.chromium.org/viewvc/chrome/trunk/src/chrome/common/pref_names.cc?view=markup

Reply:
Chromium preferences are confusing in their nature, would it be less confusing if we clarified these are for advanced use only?

There's not really a simple way to explain it. In Chromium, preferences are just internal state saved between launches, so that documentation is actually the simplest docs entrypoint I could find regarding them.

Reply:
I added a warning note at top of browser preferences saying:

>Browser preferences are available for advanced usage and should not be required in most cases.